src/OrchardCore.Modules/OrchardCore.MiniProfiler/Manifest.cs

```diff
@@ -4,7 +4,20 @@
     Name = "Mini Profiler",
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
-    Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Adds the Mini Profiler widget to pages.",
-    Category = "Performance"
+    Version = ManifestConstants.OrchardCoreVersion
 )]
+
+[assembly: Feature(
+    Id = "OrchardCore.MiniProfiler",
+    Name = "Mini Profiler",
+    Category = "Performance",
+    Description = "Adds the Mini Profiler widget to pages."
+)]
+
+[assembly: Feature(
+    Id = "OrchardCore.MiniProfiler.Admin",
```

Comment:
A short note on this would be useful on the [corresponding docs page](https://docs.orchardcore.net/en/dev/docs/reference/modules/MiniProfiler/).